cmd/podman/root.go

```diff
@@ -112,6 +113,27 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 
 	cfg := registry.PodmanConfig()
 
+	if cmd.Flag("cpu-profile").Changed {
+		f, err := os.Create(cfg.CPUProfile)
+		if err != nil {
+			return err
+		}
+		if err := pprof.StartCPUProfile(f); err != nil {
+			return err
+		}
+	}
+	if cmd.Flag("memory-profile").Changed {
+		// Same value as the default in github.com/pkg/profile.
```

Comment:
Shouldn't there be a similar os.Create(cfs.MemoryProfile) be here as well?

Reply:
That's happening below in the pre-stop hook. The CPU profile is created at the beginning an continuously being written to while the memory profile is created at the very end with all samples.

Reply:
What happens if the file can not be created?  I would prefer to know early rather then run my entire test and have it blow up in the end?

Reply:
It'll blow up at the end.  There's not much we can do, since the memory profile must be written at the end.  It's just how the stdlib wants it.

Note that the flag is hidden and meant to be only used by the devs.

Reply:
Ok.
LGTM